pact4s/README.md

```diff
@@ -0,0 +1,51 @@
+c# pact4s [Under construction]
+
+pact4s is used for contract testing.
+
+# Dependencies
+
+```scala
+  val pact4sDependencies = Seq(
+    pact4sScalaTest,
+    pact4sCirce,
+    http4sEmberClient,
+    http4sDsl,
+    http4sEmberServer,
+    http4sCirce,
+    circeCore,
+    typelevelCat,
+    scalaTest
+  )
+
+lazy val pact4s = project.in(file("pact4s"))
+  .settings(pact4sSettings)
+  .dependsOn(http % "test->test;compile->compile")
+```
+
+## Building and running contract tests
+Clone the repo.
+```
+$ git clone https://github.com/broadinstitute/cromwell.git 
+$ cd cromwell
+```
+
+If you are already using OpenJDK 17, run the following command. 
+```
+$ sbt "project pact4s" clean test  
+```
+
+Otherwise, you can run the command inside a docker container with OpenJDK 17 installed. 
+This is especially useful when automating contract tests in a GitHub Action runner which does not guarantee the correct OpenJDK version.
+```
+docker run --rm -v $PWD:/working \
+                -v jar-cache:/root/.ivy \
+                -v jar-cache:/root/.ivy2 \
+                -w /working \
+                sbtscala/scala-sbt:openjdk-11.0.16.1_1.5.5_2.13.9 \
```

Comment:
Got below error when pulling `sbtscala/scala-sbt:openjdk-11.0.16.1_1.5.5_2.13.9`

```
docker pull sbtscala/scala-sbt:openjdk-11.0.16.1_1.5.5_2.13.9
Error response from daemon: manifest for sbtscala/scala-sbt:openjdk-11.0.16.1_1.5.5_2.13.9 not found: manifest unknown: manifest unknown
```

but `docker pull sbtscala/scala-sbt:openjdk-11.0.16_1.8.1_3.2.1` works. 

Please change `openjdk-11.0.16.1_1.5.5_2.13.9` to `openjdk-11.0.16_1.8.1_3.2.1`.

Reply:
Also please update references to OpenJDK 17 to 11.